File: Modul7/Soal5/PRAK605-2210817110008-Aditya Oktaviari.c
#include <stdio.h>

int main()
{
    int ordo, nilai, hasil, i, j, a;
    scanf("%d", &ordo);
    int matriks1[ordo][ordo], matriks2[ordo][ordo];
    printf("\nMatriks A\n");

    for(i=0; i<ordo; i++){
        for(j=0; j<ordo; j++){
            scanf("%d", &nilai);
            matriks1[i][j]=nilai;
        }
    }
    printf("\nMatriks B\n");

    for(i=0; i<ordo; i++){
        for(j=0; j<ordo; j++) {
            scanf("%d", &nilai);
            matriks2[i][j]=nilai;
        }
    }
    printf("\nMatriks AxB\n");

    for(i=0; i<ordo; i++){
        for(j=0; j<ordo; j++){
            for(a=0, hasil=0; a<ordo; a++){
                hasil += matriks1[i][a]*matriks2[a][j];
            }
            printf("%d ", hasil);
        }
        printf("\n");
    }
    return 0;
}

File: Modul7/Soal5/PRAK605-2210817110008-Aditya Oktaviari.py
ordo= int(input())
print("Matriks A")
matriks1= []  
matriks2= []
for a in range(0, ordo):
    nilai1= list(map(int, input().split()))
    matriks1.append(nilai1[0 : ordo])

print("\nMatriks B")
for i in range(0, ordo):
    nilai2= list(map(int, input().split()))
    matriks2.append(nilai2[0 : ordo])
print("\nMatriks AxB")

for i in range(0, ordo):
    for j in range(0, ordo):
        hasil= 0
        for c in range(0, ordo):
            hasil += matriks1[i][c]*matriks2[c][j]
        print(hasil, end=' ')
    print()